guides-for-using-cinchy/additional-guides/cinchy-secrets-manager.md

```diff
@@ -2,95 +2,179 @@
 description: This page outlines the Cinchy Secrets Manager, added to the platform in v5.7.
 ---
 
-# Cinchy Secrets Manager
+# Cinchy secrets manager
 
 ## Overview
 
-The Cinchy platform comes with an out-of-the-box way to store secrets: the Cinchy Secrets Table. Adhering to Cinchy’s Universal Access Controls, you can use this table as a key vault (such as Azure Key Vault or AWS Secrets Manager) to store sensitive data only accessible to the users or user groups that you give access to.
+The Cinchy platform provides a built-in solution for securely storing secrets known as the Cinchy Secrets Table. Built with adherence to Cinchy’s Universal Access Controls, this table functions as a key vault similar to services like Azure Key Vault or AWS Secrets Manager. It allows you to store sensitive data that's accessible only to specific users or user groups with authorized access.
```

Comment:
I could replace access with entitlements.

Reply:
It's fine to keep it, but it's actually only **user groups**